### ISSUE TITLE
Search results

### ISSUE THREAD

**Issue by charlielafosse** [fac-15/schmidt-resources-ltd#17]
If I search for: Type: Human, Skill: CSS, Level: Medium

I get a list of what I think is the whole database?

I just want a hooman

I like the functionality of the dropdowns though!

**Comment by charlielafosse** [comment]
+ this seems to be the case for a variety of search combinations

**Comment by boi87** [comment]
yes charlie, the search functionality still needs to be implemented. So far the entire database is being displayed regardless of what you need :confused: 
Working on it!
